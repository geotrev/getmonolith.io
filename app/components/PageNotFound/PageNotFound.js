import React from "react"

import SetMeta from "app/components/SetMeta"

const PageNotFound = () => {
  return (
    <div className="has-p-xl has-text-center">
      <SetMeta pageNotFound={true} />
      <h1>{"Sorry, that page doesn't exist. :("}</h1>
      <p className="has-no-m-block-end">{"Did you enter the URL correctly?"}</p>
    </div>
  )
}

export default PageNotFound
