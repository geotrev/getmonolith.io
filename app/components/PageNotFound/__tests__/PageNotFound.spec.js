import React from "react"
import PageNotFound from "../PageNotFound"

jest.mock("app/components/SetMeta", () => global.simpleMock("SetMeta"))

describe("<PageNotFound />", () => {
  let wrapper
  beforeAll(() => {
    wrapper = mount(<PageNotFound />)
  })

  it("renders", () => {
    expect(wrapper).toMatchSnapshot()
  })
})
