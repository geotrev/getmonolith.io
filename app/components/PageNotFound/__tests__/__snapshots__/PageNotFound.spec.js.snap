// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound /> renders 1`] = `
<PageNotFound>
  <div
    className="has-padding-4 has-text-center"
  >
    <SetMeta
      pageNotFound={true}
    />
    <h1>
      Sorry, that page doesn't exist. :(
    </h1>
    <p
      className="has-no-margin-bottom"
    >
      Did you enter the URL correctly?
    </p>
  </div>
</PageNotFound>
`;
